src/course/config.json

```diff
@@ -77,6 +77,35 @@
     "_limitToSelector": ""
   },
   "build": {
-    "strictMode": true
+    "strictMode": true,
```

Comment:
I'm not sure about adding these in the default course. They shouldn't be necessary in course where all of the plugins are to be compiled. The vanilla course is used a lot for testing new plugins and these would now newly need to be updated by hand when doing that.

Reply:
> I'm not sure about adding these in the default course. They shouldn't be necessary in course where all of the plugins are to be compiled. The vanilla course is used a lot for testing new plugins and these would now newly need to be updated by hand when doing that.

I've removed these again, but as a result our PR test runner (and scheduled one) don't actually know which plugins to run tests against. There isn't any reference point for them to check so the only automated tests are those in the FW test folder

Reply:
Wider glob if no build includes or empty build.includes?

Reply:
That did the trick. Thanks Oli